### PR TITLE
Add worker build to Webpack - NEWRELIC-4383 NR-56548

### PR DESCRIPTION
**Pull request by metal-messiah**
### Overview 
This PR adds a build to the `webpack.config.js` file that generates a specific build for running inside Workers. The worker build only has relevant features (AJAX, JSErrors, Metrics, & PageActions).  This branch has naive solutions for the SendRum activateFeatures() call problem, and the onWindowLoad problem.  These will need to be evaluated, and either kept or refined.

### CODE REVIEW

File: packages/browser-agent-core/features/page-view-timing/instrument/index.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export class Instrument extends FeatureBase {
     }
 
     // first interaction and first input delay
-    if ('addEventListener' in document) {
+    if (typeof document !== 'undefined' && 'addEventListener' in document) {
```
[the remark this answers is not in the document]
Comment (metal-messiah) [reply packages/browser-agent-core/features/page-view-timing/instrument/index.js:49]: This was throwing an error in the worker

[the remark this answers is not in the document]
Comment (metal-messiah) [reply packages/browser-agent-core/features/page-view-timing/instrument/index.js:49]: merge remnant

File: packages/browser-agent-core/features/jserrors/aggregate/index.js [hunk 2]
```diff
@@ -19,6 +19,8 @@ import { now } from '../../../common/timing/now'
 
 import { FeatureBase } from '../../../common/util/feature-base'
 
+const ref = typeof window !== 'undefined' ? window : self
```
[the remark this answers is not in the document]
Comment (metal-messiah) [reply packages/browser-agent-core/features/jserrors/aggregate/index.js:22]: merge remnant

File: cdn/agent-loader/worker.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { detectPolyfillFeatures } from './utils/feature-detection'
+const PolyfillFeatures = detectPolyfillFeatures();
```
[the remark this answers is not in the document]
Comment (metal-messiah) [reply cdn/agent-loader/worker.js:6]: fair

File: packages/browser-agent-core/features/jserrors/aggregate/index.js [hunk 4]
```diff
@@ -146,6 +147,7 @@ export class Aggregate extends FeatureBase {
   }
 
   storeError(err, time, internal, customAttributes) {
+    console.log("storeError...", err)
```
[the remark this answers is not in the document]
Comment (metal-messiah) [reply packages/browser-agent-core/features/jserrors/aggregate/index.js:150]: merge remnant

**Review by cwli24 — APPROVED** [review]

**Review by cwli24 — APPROVED** [review]

**Pull request merged**